app/views/passengers/_check_existing.haml

```diff
@@ -0,0 +1,9 @@
+.modal-dialog{role: 'document'}
+  .modal-content
+    .modal-header
+      %h5 A passenger already exists for this spire ID. Go to edit that passenger?
+      %button.close{data: {dismiss: 'modal', 'aria-label':'Close'}}
+        %span{'aria-hidden': 'true'} &times;
+    .modal-footer
+      %button.btn.btn-default{data: {dismiss: 'modal', 'aria-label':'Close'}} Keep adding new
+      = link_to 'Edit existing passenger', edit_passenger_url(@passenger), class: 'btn btn-primary'
```

Comment:
Some verb agreement / parallel construction nit pick. Perhaps:

*  A passenger already exists for this Spire ID. Edit that passenger? (Spire should be capitalized)
*  Add new passenger
*  Edit existing passenger